### PR TITLE
Add new strings and tags to internet-health.lang

### PR DESCRIPTION
**Pull request by peiying2**

### CODE REVIEW

**Comment by peiying2** [comment]
Changes made to:
- internet-health page per bug  1319909
- firefox/channel/index page per bug 1320098

File: en-US/firefox/channel/index.lang [hunk 1]
```diff
@@ -99,6 +100,12 @@ Build, test, scale and more with the only browser built just for developers.
 Firefox Developer Edition automatically sends feedback to Mozilla.
 
 
+## TAG: android_nightly_copy
+;Check out new Android features in their earliest stages. Enjoy at your own risk.
+Check out new Android features in their earliest stages. Enjoy at your own risk.
+
+# This string is obsolete. Do not remove.
+# This string is used as a fallback.
```
Comment (flodolo) [inline en-US/firefox/channel/index.lang]: Remove both these comments. In this case the string is already (and still) used for desktop, is used a fallback only for Android, no need to tell localizers.

Comment (flodolo) [inline en-US/firefox/channel/index.lang]: Also, makes sure to leave two empty lines between strings.

**Comment by flodolo** [comment]
This looks good. At this point you need to update all locales, adding the new strings (don't forget you changed two files). Commit and push.

Then follow these instructions to import existing translations
https://github.com/mozilla-l10n/documentation/blob/master/mozilla.org/import_translations.md

We want to split adding strings and importing them in two commits, to make the diff easier to check.

**Pull request merged**